qutebrowser/completion/models/urlmodel.py

```diff
@@ -67,6 +68,7 @@ def url(*, info):
     quickmarks = [(url, name) for (name, url)
                   in objreg.get('quickmark-manager').marks.items()]
     bookmarks = objreg.get('bookmark-manager').marks.items()
+    buffers = utils.list_buffers()
```

Comment:
If this sticks around, there's no reason to do this computation for everyone, we only need to do it if buffers is in completions.

Reply:
Done.